Automattic/portkey-android
# Story creation: bottom strip view - tap listeners (part 5)

Builds on #238 

This PR implements the listeners for handling taps on the bottom strip (plus icon button and Story frames).

To test:
1. open the app
2. capture some image
3. observe it's inserted as the first image in the bottom strip
4. capture another image
5. observe it's inserted as the first image in the bottom strip, displacing the pre-existing one.
6. do as many times as you wish.

Select:
Once you have captured several (at least 2) images, test changing between thems as follows:
1. tap on an image
2. observe the background image being presented in the editor is indeed the selected iimage.

![click](https://user-images.githubusercontent.com/6597771/70874724-15d22000-1f92-11ea-88cf-e8461a23290e.gif)
